### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Ahmad Jano

### DISCUSSION THREAD

**Discussion by ahmadjano** [heig-vd-dai-course/heig-vd-dai-course#92] [Show and tell]
Hello, the main idea of my project is to extract metadata information from picture files, mainly GPS coordinates, and then generate plain text files containing these coordinates (possibly in a form of google maps links).

For the time being I'm thinking of using an external library ([javaxt](https://www.javaxt.com/javaxt-core/io/Image)) to obtain the GPS information but the generation of the text files is done by using the native Java IO library. 

[Link to the repo](https://github.com/ahmadjano/dai-practical-work-1)

**Comment by ludelafo** [comment]
Great, thank you! Looking forward to your project.

**Comment by ahmadjano** [comment]
Last commit: [af5db71](https://github.com/ahmadjano/dai-practical-work-1/commit/af5db7149a936ec2738ee3a38ba0a407025a4d38)
